pkg/config/validator.go

```diff
@@ -360,6 +363,22 @@ func validatePluginRBAC[P pluginProvider](sl validator.StructLevel, pluginConfig
 	}
 }
 
+func validateActionExecutors(sl validator.StructLevel, executors map[string]Executors, bindings []string) {
+	for _, executor := range bindings {
+		execConf := executors[executor]
+		for pluginKey, plugin := range execConf.Plugins {
+			if !plugin.Enabled {
+				continue
+			}
+			if plugin.Context.RBAC != nil {
+				if plugin.Context.RBAC.Group.Type == ChannelNamePolicySubjectType {
+					sl.ReportError(bindings, pluginKey, executor, invalidActionRBACTag, "")
+				}
+			}
```

Comment:
we can easily get rid of nested `if's`
```suggestion
			if plugin.Context.RBAC == nil {
				continue
			}
			if plugin.Context.RBAC.Group.Type == ChannelNamePolicySubjectType {
				sl.ReportError(bindings, pluginKey, executor, invalidActionRBACTag, "")
			}
```

BTW, maybe this should be a part of `validateExecutorBindings`? Just a suggestion but not pushing into it 👍

Reply:
`validateExecutorBindings` is also used by `botBindingsStructValidator`, so I opted for a brand new func.